### PR TITLE
add the possiblity to use a contributors team instead of authors with years

### PR DESCRIPTION
**Pull request by TiKeil**
Putting 
`contributors_team = ' someone ' `
in the config gives are contributors team instead of separated Authors.

### CODE REVIEW

File: pylicense3/cli.py [hunk 1]
```diff
@@ -183,17 +183,20 @@ def write_header(
...
-        max_author_length = max(max_author_length, len(author))
-    for author in sorted(authors.keys()):
-        year = '(' + authors[author] + ')'
-        if len(prefix) + 4 + max_author_length + len(year) <= max_width:
-            for ii in range(max_author_length - len(author)):
-                author += ' '
-        target.write(u'{}   {} {}\n'.format(prefix, author, year))
+    # authors, either as dict with years or only a contribution team
+    if isinstance(authors, str):
+        target.write(prefix + f' Contributors team: {authors}\n')
```
Comment (renefritze) [inline pylicense3/cli.py:188]: What do you think about giving the user even more flexibility here? 
```suggestion
        target.write(f' {prefix} {authors}\n')
```
So not even the "Contributors team" would be hard coded.

File: pylicense3/cli.py [hunk 2]
```diff
@@ -235,8 +238,11 @@ def process_file(filename, config, root):
     prefix = getattr(config, 'prefix', '#')
     lead_out = getattr(config, 'lead_out', None)
     lead_in = getattr(config, 'lead_in', None)
-    # read authors and respective years
-    authors = get_authors(filename, root)
+    if hasattr(config, 'contributors_team'):
+        authors = getattr(config, 'contributors_team')
+    else:
+        # read authors and respective years
+        authors = get_git_authors(filename, root)
```
Comment (renefritze) [inline pylicense3/cli.py:245]:
```suggestion
    authors = getattr(config, 'contributors_team', get_git_authors(filename, root)
```

**Comment by renefritze** [comment]
I'll have a look at the breaking CI tomorrow. Most likely cause I transferred the repo today.

File: pylicense3/cli.py [hunk 3]
```diff
@@ -235,8 +238,7 @@ def process_file(filename, config, root):
     prefix = getattr(config, 'prefix', '#')
     lead_out = getattr(config, 'lead_out', None)
     lead_in = getattr(config, 'lead_in', None)
-    # read authors and respective years
-    authors = get_authors(filename, root)
+    authors = getattr(config, 'contributors_team', get_git_authors(filename, root)
```
Comment (TiKeil) [inline pylicense3/cli.py:241]:
```suggestion
    authors = getattr(config, 'contributors_team', get_git_authors(filename, root))
```

**Comment by TiKeil** [comment]
Thanks for your comments !

**Review by renefritze — CHANGES_REQUESTED** [review]
I think you applied the suggestion with my missing end paren.

**Review by renefritze — APPROVED** [review]
Disregard previous review, you already fixed my mistake :)

**Comment by TiKeil** [comment]
Yes. I will add another commit regarding the user flexibility of the team name

**Comment by renefritze** [comment]
> Yes. I will add another commit regarding the user flexibility of the team name

Great. Feel free to merge and tag for release then. Thanks!

**Pull request merged**